src/utils/update/trust.model.fetch.ts

```diff
@@ -97,10 +97,10 @@ const fetchAndMapIndivdualTrustees = async (
   overseasEntityId: string,
   trust: Trust
 ) => {
-  const individualTrustees = await getIndividualTrustees(req, transactionId, overseasEntityId, trust.trust_id);
+  const individualTrustees = await getIndividualTrustees(req, transactionId, overseasEntityId, trust.ch_reference ?? trust.trust_id);
```

Comment:
Is the trust id needed here at all? (And in the other appearances in this file)

Reply:
It's a fall back ch_reference should never be undefined but maybe better to be more careful about that.

Reply:
I'll address on https://github.com/companieshouse/overseas-entities-web/pull/1146